crates/db_views/src/comment_view.rs

```diff
@@ -253,6 +253,11 @@ fn queries<'a>() -> Queries<
       limit_and_offset(options.page, options.limit)?
     };
 
+    // distinguished comments should go first when viewing post
+    if options.post_id.is_some() || options.parent_path.is_some() {
+      query = query.order_by(comment::distinguished.desc());
```

Comment:
`then_order_by`

Reply:
Fixed. Seems like it would be a good idea to disallow order_by in general.